wetterdienst/file_path_handling/file_list_creation.py

```diff
@@ -48,3 +50,34 @@ def create_file_list_for_dwd_server(
     ]
 
     return file_index[DWDMetaColumns.FILENAME.value].values.tolist()
+
+
+def create_filepath_for_radolan(
+        date_time: datetime,
+        time_resolution: TimeResolution
```

Comment:
Are you checking for valid TimeResolutions before?

Reply:
I do in collect_radolan_data and in DWDRadolanRequest